types/target.go

```diff
@@ -43,16 +45,16 @@ func (x Target) Cmp(y Target) int {
 }
 
 // Difficulty returns the difficulty associated with a given target.
-func (t Target) Difficulty() Currency {
-	if t == (Target{}) {
+func (x Target) Difficulty() Currency {
+	if x == (Target{}) {
```

Comment:
these should use `t` throughout instead of `x`

Reply:
So for functions like this:

func (x Target) SubtractDifficulties(y Target) (t Target)

this needs to be changed to:

func (t Target) SubtractDifficulties(y Target) (x Target)

but does this also mean that callers of this function have to change as well now that the return variable changes? or that doesn't really matter?

It's complaining about duplicate argument if I leave them both as t.

Reply:
golint just wants the name of the receiving variable to be consistent across methods. So it should be `func (t Target) Foo` for every method. Callers of the function shouldn't be affected.